Runtime/Model/JsonData/Annotations.cs

```diff
@@ -40,7 +40,12 @@ public Annotations(Exception exception, int gameObjectDepth)
             var environmentVariables = new Dictionary<string, string>();
             foreach (DictionaryEntry variable in Environment.GetEnvironmentVariables())
             {
-                environmentVariables.Add(variable.Key.ToString(), Regex.Escape(variable.Value.ToString() ?? "NULL"));
+                if (variable.Key == null)
```

Comment:
I don't see this bug fix in release notes. Also, how does one disable environment variables?

Reply:
> I don't see this bug fix in release notes.
Adding info to changelog - I focused there only on information that impacts users.
>  Also, how does one disable environment variables?
Info is available here:
https://github.com/backtrace-labs/backtrace-unity#data-privacy